pennylane/transforms/split_non_commuting.py

```diff
@@ -216,6 +304,48 @@ def circuit(x):
     .. details::
         :title: Usage Details
 
+        **Shot distribution**
+
+        With finite-shot measurements, the default behaviour of ``split_non_commuting``
+        will use 1 execution with ``shots`` each for each group of commuting terms. With the
```

Comment:
```suggestion
        With finite-shot measurements, the default behaviour of ``split_non_commuting``
        is to perform one execution with the total number of ``shots`` for each group of commuting terms. With the
```

@albi3ro Maybe even better now?